backend/backup_media.js

```diff
@@ -130,7 +150,7 @@ const processBatch = async (startWithId) => {
         console.log(outputPath, 'is already there');
         saveEntry(mediaType, outputPath, outputFilename, mediaSrc, itemType, itemId);
       } else {
-        console.log(`Processing media item `, mediaSrc);
+        console.log(`Processing media item `, hideUrl(mediaSrc));
```

Comment:
I guess this is to hide real media URLs in the log so that they are not visible in the GH flow log? Maybe just remove this log entry then?

Reply:
>  this is to hide real media URLs in the log so that they are not visible in the GH flow log

Correct. Wanted to have a compromise between the info logged and some degree of privacy/anonymity. I think logging  `content_id` and the domain of the media URL would be sufficient.

Long term solution would be to create another private repo in the `spaceshelter` org and run this workflow from there.

Reply:
I see, makes sense. Also need to handle log entry on the line 39 in the `download` function:
```
      console.log(`Wrote ${responseBinary.length} bytes to ${dest} for ${url}`);
```